src/lib/utils.ts

```diff
@@ -0,0 +1,6 @@
+import { ClassValue, clsx } from "clsx";
+import { twMerge } from "tailwind-merge";
+
+export function cn(...inputs: ClassValue[]) {
+  return twMerge(clsx(inputs));
+}
```

Comment:
Use an arrow function, also please rename this function to be descriptive.

```suggestion
export const mergeClasses(...inputs: ClassValue[]) => twMerge(clsx(inputs));
```

Reply:
This part is a copy and past
from Aceternity ui, I didn't end up using it but I just set it up in case it could be used: For example
https://ui.aceternity.com/components/macbook-scroll